src/main.rs

```diff
@@ -9,7 +9,11 @@ use std::cell::RefCell;
 use std::error::Error;
 use std::fmt;
 use std::str::FromStr;
-use tonic_lnd::{Client, ConnectError};
+use tonic_lnd::{tonic::Code, Client, ConnectError};
+
+const ONION_FEATURE_BIT: u32 = 39;
+const SERVICE_UNIMPLEMENTED: &str = "Peer or sign service is unimplemented. Remember to
```

Comment:
Can this specifically be a `PEER_SERVICE_UNIMPLEMENTED` because we'll know what rpc server we're missing when we make specific calls and they fail with `Unimplemented`. 

Also, wdyt about having an `ErrUnimplemented` enum that implements `Error` instead of a const? Not sure what's more idiomatic rust, but enums seem to be all the rage.

Reply:
Yes I like this idea, especially since there will be a few... signerrpc, peersrpc, dev and maybe others